ethereum/api/src/test/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/AbstractEngineForkchoiceUpdatedTest.java

```diff
@@ -442,7 +451,196 @@ public void shouldIgnoreUpdateToOldHeadAndNotPreparePayload() {
     verify(engineCallListener, times(1)).executionEngineCalled();
   }
 
-  private EngineUpdateForkchoiceResult assertSuccessWithPayloadForForkchoiceResult(
+  @Test
+  public void shouldReturnInvalidIfPayloadTimestampIsInvalid() {
```

Comment:
In this case, how is the timestamp invalid? Perhaps more descriptive test name to make it clearer why it's invalid. Is this testing the case where ExecutionPayloadV1 is used when timestamp < shanghai?

Reply:
yeh makes sense, it's the existing fcu validation, not new for withdrawals, i.e. whether or not the timestamp is in the future w.r.t head